sds_data_manager/lambda_code/IAlirtCode/ialirt_ingest.py

```diff
@@ -27,23 +28,43 @@ def lambda_handler(event, context):
         and runtime environment.
 
     """
+    # TODO: these steps will be put into different functions.
     logger.info("Received event: %s", json.dumps(event))
 
-    table_name = os.environ.get("TABLE_NAME")
+    ingest_table_name = os.environ.get("INGEST_TABLE")
+    algorithm_table_name = os.environ.get("ALGORITHM_TABLE")
     dynamodb = boto3.resource("dynamodb")
-    table = dynamodb.Table(table_name)
+    ingest_table = dynamodb.Table(ingest_table_name)
+    algorithm_table = dynamodb.Table(algorithm_table_name)
 
     s3_filepath = event["detail"]["object"]["key"]
     filename = os.path.basename(s3_filepath)
     logger.info("Retrieved filename: %s", filename)
 
-    # TODO: item is temporary and will be replaced with actual packet data.
+    # TODO: Each of these steps in temporary, but provides an idea
```

Comment:
Are all the steps in the same lambda?

Reply:
That is correct. I thought about doing separate lambdas but then realized there was no reason for it.